App.js

```diff
@@ -3,10 +3,17 @@ import {Provider} from 'react-redux';
 import {createStore, applyMiddleware} from 'redux';
 import ReduxThunk from 'redux-thunk';
 import firebase from 'firebase';
+import {TabNavigator, StackNavigator} from 'react-navigation';
 
 import {FIREBASE_CONFIG} from './env';
 import reducers from './src/reducers';
-import Router from './src/Router';
+
+import LoginScreen from './src/screens/LoginScreen';
+import SearchScreen from './src/screens/SearchScreen';
+import FilterScreen from './src/screens/FilterScreen';
+import FavoritesScreen from './src/screens/FavoritesScreen';
+import AddSiteScreen from './src/screens/AddSiteScreen';
+import MoreScreen from './src/screens/MoreScreen';
```

Comment:
No longer importing `Router`, and instead bringing in each individual screen. I'll probably actually end up refactoring this back out to its own `Router` file, but as of right now `Router` still includes all of the `react-native-router-flux` work.